transform/mattermost-analytics/models/marts/nps/_nps__models.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+
+models:
+  - name: dim_nps_feedback
+    description: |
+      List of known customers with active on prem licenses.
+
+    columns:
+      - name: server_id
+        description: The ID of the server the event originated from..
+        tests:
+          - not_null
+      - name: user_id
+        description: The ID of the user that sent the event.
+      - name: license_id
+        description: The license id.
+      - name: feedback
+        description: NPS feedback submitted.
+      - name: score
+        description: NPS score submitted.
+      - name: server_version
```

Comment:
Perhaps this can be a version id and it can act as a reference to `dim_version` that is already defined.

Reply:
Will do this in the next iteration of this code, as this is not a priority at the moment.